clients/java/src/main/java/io/camunda/zeebe/client/ZeebeClient.java

```diff
@@ -592,4 +593,29 @@ static ZeebeClientCloudBuilderStep1 newCloudClientBuilder() {
    */
   @ExperimentalApi("https://github.com/camunda/camunda/issues/20596")
   UserTaskQuery newUserTaskQuery();
+
+  /**
+   * Executes a search request to query decision definitions.
+   *
+   * <pre>
+   * long decisionDefinitionKey = ...;
+   *
+   * zeebeClient
+   *  .newDecisionDefinitionQuery()
+   *  .filter((f) -> f.key(decisionDefinitionKey))
+   *  .sort((s) -> s.name().asc())
```

Comment:
💭 Similar the previous one. 

```suggestion
   *  .sort((s) -> s.dmnDecisionName().asc())
```